shared/actions/wallets.tsx

```diff
@@ -1122,10 +1122,11 @@ type AirdropDetailsJSONType = {
 } | null
 
 const updateAirdropDetails = (
-  _: TypedState,
+  state: TypedState,
   __: WalletsGen.UpdateAirdropStatePayload | ConfigGen.DaemonHandshakeDonePayload,
```

Comment:
nit: `__` could become `_`

Reply:
Thanks!  I wonder if there's an eslint rule for validating uses of underscore placeholders :)